.github/workflows/ci.yaml

```diff
@@ -860,30 +855,11 @@ jobs:
       - name: Set up Google Cloud SDK
         uses: google-github-actions/setup-gcloud@v2
 
-      - name: Setup Node
-        uses: ./.github/actions/setup-node
-
-      - name: Setup Go
-        uses: ./.github/actions/setup-go
-
-      - name: Install goimports
-        run: go install golang.org/x/tools/cmd/goimports@latest
-      - name: Install nfpm
-        run: go install github.com/goreleaser/nfpm/v2/cmd/nfpm@v2.16.0
-
-      - name: Install zstd
-        run: sudo apt-get install -y zstd
-
-      - name: Build Release
-        run: |
-          set -euo pipefail
-          go mod download
-
-          version="$(./scripts/version.sh)"
-          make gen/mark-fresh
-          make -j \
-            build/coder_"$version"_windows_amd64.zip \
-            build/coder_"$version"_linux_amd64.{tar.gz,deb}
+      - name: Upload build artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: coder
+          path: ./build
```

Comment:
Are you sure this doesn't extract to `./build/build/*` instead of `./build/*`? The entries are added from` ./build/*` in the upload artifact step

Reply:
This is just the path on where to extract. I am going to test it and then see if I need to change it.